app/controllers/build_commands_controller.rb

```diff
@@ -10,14 +10,10 @@ def show
 
   def update
     command = params[:command][:command]
-    if command.blank?
```

Comment:
... just do `destroy!` and let it blow up ... user should have seen the warning ... also don't need js response ...